app/kumactl/cmd/root.go

```diff
@@ -51,6 +52,11 @@ func NewRootCmd(root *kumactl_cmd.RootContext) *cobra.Command {
 			// once command line flags have been parsed,
 			// avoid printing usage instructions
 			cmd.SilenceUsage = true
+
+			if args.noConfig {
+				return nil
```

Comment:
I'm a bit anxious this can lead to bugs. We assume in kumactl that there is a config. Wouldn't it make sense to provide default configuration? Essentially, put
```
root.Runtime.Config = kumactl_config.DefaultConfiguration()
```
before return

Reply:
ack, on it